fetch_firmware.sh.m4

```diff
@@ -32,7 +32,9 @@ FW_VERSION=${1:-M4_FW_VERSION}
 NAP_VERSION=${2:-M4_NAP_VERSION}
 
 CCACHE_S3_PATH=s3://swiftnav-artifacts/piksi_buildroot/$BR_VERSION
-FW_S3_PATH=s3://M4_BUCKET/piksi_firmware_private/$FW_VERSION/v3
+FW_S3_BUCKET=${3:-M4_FW_S3_BUCKET}
+FW_S3_PREFIX=${4:-M4_FW_S3_PREFIX}
+FW_S3_PATH=s3://M4_BUCKET/$FW_S3_PREFIX/$FW_VERSION/v3
```

Comment:
Can this be moved up with the other passed in substitutions?

```
FW_VERSION=${1:-M4_FW_VERSION}
NAP_VERSION=${2:-M4_NAP_VERSION}
FW_S3_BUCKET=${3:-M4_FW_S3_BUCKET}
FW_S3_PREFIX=${4:-M4_FW_S3_PREFIX}
```

Reply:
Suggested change implemented in commit 876cfd03311dfc201a068d41d2b5cfa2bbecb1a9

Reply:
I'm not able to find this commit, is it in this repo?